3-deployment-targets-adapters-and-stacks/using-remix-stacks/package-lock.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "indie-stack-template",
+  "name": "using-remix-stacks-4ce2",
```

Comment:
FYI: this change got picked up due to a mismatch in the `package-lock.json` and the `package.json` file. Figured it might be worth fixing now 🤷